src/components/Layout.vue

```diff
@@ -0,0 +1,16 @@
+<template>
+  <div>
```

Comment:
- The component should include this
```<Container :slim="true">```
(And Container should be removed from the views if using Layout)
- The breadcrumb back / home / space link can be moved inside the Layout in the "content-left" or "content-right" slot